azure-pipelines.yml

```diff
@@ -18,10 +18,10 @@ pr:
     include:
     - '*'  # must quote since "*" is a YAML reserved character; we want a string
 
-# the version of OpenBLAS used is currently 0.3.8.dev
+# the version of OpenBLAS used is currently 0.3.15
```

Comment:
comment looks like version typo, didn't check if still on master, not super important for now

Reply:
yes, I believe that should be `17`